pitch.md

```diff
@@ -1,44 +1,90 @@
-# Project Title (replace me)
+# CaffeineScript
 
 ## Overview
 _What's this project about, briefly?_
 
+This project is about creating a language to specify coffee/alcoholic drinks, so they can be made to your taste by an automated machine. We already have some automated coffee machines, but these machines usually have a limited number of adjustable settings and are considered by many to be inferior to an actual barista's work for this reason. Theoretically, if a project like this was wildly successful it'd eliminate the need for bartenders/baristas.
 
 ## The user and a language
 This section describes who the project would serve and why a language might be a
 good way to meet their needs.
 
```

Comment:
This is important to think about. User of the language vs user of the coffee machine